packages/store-sync/src/fromEventSource.ts

```diff
@@ -0,0 +1,10 @@
+import { Observable } from "rxjs";
+
+export function fromEventSource<T>(url: string | URL): Observable<MessageEvent<T>> {
+  return new Observable<MessageEvent>((subscriber) => {
+    const eventSource = new EventSource(url);
+    eventSource.onmessage = (ev): void => subscriber.next(ev);
+    eventSource.onerror = (ev): void => subscriber.error(ev);
```

Comment:
I think `onerror` gives you an `Event`, not an `Error`, so I wonder if it's a bad idea to `subscriber.error` with an `Event`. Is there an error embedded in the event we can use instead? or turn it into some error?